scripts/canary-release.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+set -e
+
+function log() {
+  echo '\033[36m[canary-release]\033[0m' "$@"
+}
+
+function fail() {
+  echo '\033[31mFAILURE:\033[0m' "$@"
+}
+
+log "Preparing for canary release..."
+
+log "Installing dependencies..."
+if ! npm i; then
+  fail "npm install failed."
+  exit 1
+fi
+echo ""
+
+log "Running pre-release script..."
+if ! ./scripts/pre-release.sh; then
+  fail "Pre-release script failed"
+  exit 1
+fi
+
+log "Bumping up MDC Web version to canary..."
+if ! npx lerna version premajor --no-git-tag-version --no-push --preid canary.$(git rev-parse --short HEAD) --yes; then
+  fail "lerna version command failed"
+  exit 1
+fi
+echo ""
+
+PREVIOUS_GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
FWIW, bash best practice (and Google's style guide) recommends using `lower_snake_case` for bash variables. `UPPER_SNAKE_CASE` is typically meant for environment variables

Reply:
I'll update this in my next PR. Thanks!